### PR TITLE
10.17-releasenotes-review

### PR DESCRIPTION
**Pull request by dianaolives**

### CODE REVIEW

File: content/release-10-17-0/datahub-10-17-0-bundle/10_17_0.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ The look and feel of the UI has been updated in line with the Cumulocity IoT pla
...
-|Dremio has announced in its [Dremio 4.0 release notes](https://docs.dremio.com/release-notes/40-release-notes.html#deprecations) to deprecate some functionality on mixed types. Since Dremio 13.2, GROUP BY or DISTINCT over mixed type expressions are no longer supported. It is necessary to cast these expressions to a simple type before using them in a GROUP BY or DISTINCT clause. Also mixed type expressions cannot be used in the SELECT clause, when retrieving data over ODBC or JDBC.|
+|Dremio has announced in its [Dremio 4.0 release notes](https://docs.dremio.com/release-notes/40-release-notes.html#deprecations) **broken link** to deprecate some functionality on mixed types. Since Dremio 13.2, GROUP BY or DISTINCT over mixed type expressions are no longer supported. It is necessary to cast these expressions to a simple type before using them in a GROUP BY or DISTINCT clause. Also mixed type expressions cannot be used in the SELECT clause, when retrieving data over ODBC or JDBC.|
```
Comment (dianaolives) [inline content/release-10-17-0/datahub-10-17-0-bundle/10_17_0.md:25]: @BeateRixen this link doesn't work currently and I could not really find a new one on the Dremio docs website where it talks about this specifically.

Comment (BeateRixen) [reply content/release-10-17-0/datahub-10-17-0-bundle/10_17_0.md:25]: @cheinz-sag Can you please check?

Comment (cheinz-sag) [reply content/release-10-17-0/datahub-10-17-0-bundle/10_17_0.md:25]: Hi,

great that you caught this one! Can we do it like that:
- for version 10.13 and above remove the complete entry "Dremio has announced..."
- for version 10.11 drop the link and say: Dremio has deprecated some functionality on mixed types.

If you're good with this, I'll set up corresponding branches.

Comment (BeateRixen) [reply content/release-10-17-0/datahub-10-17-0-bundle/10_17_0.md:25]: Sure. @dianaolives can you please prepare the PRs accordingly.

Comment (dianaolives) [reply content/release-10-17-0/datahub-10-17-0-bundle/10_17_0.md]: @BeateRixen @cheinz-sag I've dropped the link for 10.11 and removed the whole row on the table for 10.13 and above.

Comment (cheinz-sag) [reply content/release-10-17-0/datahub-10-17-0-bundle/10_17_0.md]: Hi Diana,

thanks! That looks good.

**Pull request merged**